.gitignore

```diff
@@ -159,7 +159,7 @@ PublishScripts/
 ## The packages folder can be ignored because of Package Restore

```

Comment:
We still do not have Paket :-)

It seems to me that the uncomment of 157-167 requires package restore and that we should remove all content of the packages folder?

IMHO this is not related to the issue of this PR, so I'll do these changes when I do the Paket stuff